content/projects/ecft-vs-pipeline.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: ECFT-VS Pipeline
+title: ECFT VS-Pipeline
```

Comment:
I just realized that it's ECFT up here and ECF-T in the text below

Reply:
I've updated the abbreviation in the text. Generally should be ECF transporters. However, to keep the title short, should it be ECFT ?

Reply:
Yes, we can do it like that.